
syntax = "proto3";

package mesop.components.select;

message SelectType {
  bool disabled = 2;
  bool disable_ripple = 3;
  double tab_index = 4;
  bool hide_single_selection_indicator = 5;
  string placeholder = 6;
  bool required = 7;
  bool multiple = 8;
  string value = 9;
  string on_select_opened_change_event_handler_id = 14;
  string on_select_selection_change_event_handler_id = 15;

  repeated SelectOption options = 16;
  string label = 17;
}

message SelectOption {
  string label = 1;
  string value = 2;
}
